docs/algorithms/kem/classic_mceliece.md

```diff
@@ -35,8 +35,8 @@
 
 |       Implementation source       | Identifier in upstream   | Supported architecture(s)   | Supported operating system(s)   | CPU extension(s) used   | No branching-on-secrets claimed?   | No branching-on-secrets checked by valgrind?   | Large stack usage?‡   |
 |:---------------------------------:|:-------------------------|:----------------------------|:--------------------------------|:------------------------|:-----------------------------------|:-----------------------------------------------|:----------------------|
-| [Primary Source](#primary-source) | clean                    | All                         | All                             | None                    | True                               | True                                           | True                  |
-| [Primary Source](#primary-source) | avx2                     | x86\_64                     | Linux,Darwin                    | AVX2,POPCNT             | False                              | True                                           | True                  |
+| [Primary Source](#primary-source) | clean                    | All                         | All                             | None                    | False                              | False                                          | True                  |
```

Comment:
Do we now understand these changes to McEliece?

Reply:
Pinging @praveksharma to take a look at this.

Reply:
`classic_mceliece.md` wasn't updated along with `classic_mceliece.yml` in #1552 (presumably I forgot to run `update_upstream_alg_docs.py` at the time) so these changes are desirable.

Reply:
Will be resolved when #1589 lands and this PR rebases to it.

Reply:
@SWilson4 Update: If this PR is rebased to #1595 (assuming you approve and we land that), `copy_from_upstream` is fixed and idempotent again, pulling in HQC in the process. CI can stay as-is.

I have tested that and could do a `git push -f` on this PR to finalize this if you want/agree. Drawback: I'd most likely become co-author to this PR and cannot approve this any more. We then need two _more_ people to approve this (hint, hint, @dstebila @bhess @praveksharma :)

Reply:
I can do the rebase/push and save people the trouble.

Reply:
Thanks. Saw it. Resolved.